### DISCUSSION TITLE
Migrate The Code To Kotlin Compose

### DISCUSSION THREAD

**Discussion by la-niina** [2dust/v2rayNG#2734] [Ideas]
Can I help migrate the code to kotlin compose but still v2rayNG as a contribution to v2rayNG. It will take me about a week to complete. If its something the community of v2ray would be into I am willing to contribute in doing it.

**Comment by 2dust** [comment]
Just do it
For better testing, you can PR function (Activity) one by one.
Start PR with simple function
